src/components/base/JAssetSymbol/jAssetSymbol.scss

```diff
@@ -15,16 +15,23 @@
   font-size: inherit;
 
   &.-symbol-not-listed::before {
-    @include font;
+    @include font(14px, 38px);
 
-    display: block;
-    content: attr(data-symbol);
     position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
+    box-sizing: border-box;
+    content: attr(data-symbol);
+    overflow: hidden;
+    width: 40px;
```

Comment:
Is it always 40px? What about Asset Picker?

Reply:
It depends on version of Asset Picker do you ask :D